offices/_vars-offices.html

```diff
@@ -2,9 +2,6 @@
 {% set query = queries.initiative %}
 {% set initiatives = query.search_with_url_arguments(size=6, filter_has_parent="false", filter_related_office=office.slug) %}
 
-{% set tag = "foia" %}
-{% set activity_type = ["cfpb_report"] %}
```

Comment:
It looks like _single.html uses the variable activity_type.  Are we even doing activity feeds for offices at this point?